handlers/stream_cache.go

```diff
@@ -0,0 +1,123 @@
+package handlers
+
+import (
+	"fmt"
+	"sync"
+)
+
+// Shared server state. Each pipeline (usually endpoint) having separate books.
```

Comment:
The same comment about comments 🙃 Please use the Golang convections for comments.

Reply:
I keep missing this..